bentoml/cli/bento_management.py

```diff
@@ -167,35 +143,37 @@ def get(bento, limit, ascending_order, print_location, labels, output):
         '--order-by', type=click.Choice(['created_at', 'name']), default='created_at',
     )
     @click.option('--ascending-order', is_flag=True)
+    @click.option('--yatai-url', type=click.STRING, help='Yatai server URL.')
```

Comment:
add more details to this `help` message? tell user this is optional, use it when using remote yatai service, with an example etc